src/components/FilterDropdown.jsx

```diff
@@ -20,20 +18,15 @@ const useStyles = makeStyles((theme) => ({
 function FilterDropdown(props) {
   const { label, items, value, removeNone, onChange } = props;
   const classes = useStyles();
-  const location = useLocation();
-  const params = queryString.parse(location.search, { parseNumbers: true });
-  const sortByParams =
-    params.sortBy === undefined ? 'deadlineAsc' : params.sortBy;
-  const [choice, setChoice] = useState(value === 'sorting' ? sortByParams : '');
 
   function selectChoice(event) {
-    setChoice(event.target.value);
     onChange(event.target.value);
   }
+
   return (
     <FormControl variant="outlined" className={classes.formControl}>
       <Select
-        value={choice}
+        value={value}
         onChange={selectChoice}
```

Comment:
You can get rid of the `selectChoice` function now.
```suggestion
        onChange={(e) => onChange(e.target.value)}
```

Reply:
Ok nice